GoogleDataLogger/GoogleDataLogger/Classes/Public/GDLLogScorer.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class GDLLogEvent;
+
+/** This protocol defines the common interface of a log scorer. Log scorers are stateful objects
```

Comment:
What exactly does `score` mean?  It seems like it's a priority, but if that's the case, why not call it `priority`?

Reply:
Renamed everything to be in terms of prioritization instead of scoring.